packages/fluent/scss/core/_variables.scss

```diff
@@ -87,6 +88,19 @@ $kendo-border-radius-sm: calc( #{$kendo-border-radius} / 2 ) !default;
 $kendo-border-radius-md: $kendo-border-radius !default;
 $kendo-border-radius-lg: calc( #{$kendo-border-radius} * 2 ) !default;
 
+// Icons
+$kendo-icon-size: 16px !default;
```

Comment:
the icon sizes can be taken from the icon component, they do not need to be defined here